observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -0,0 +1,50 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-about"]
+= About logging 6.0
+:context: logging-6x
+
+toc::[]
+
+The `ClusterLogForwarder` custom resource is the central configuration point for OpenShift Logging 6.0. It defines the desired state of your logging system, including inputs, outputs, and pipelines. The Cluster Logging Operator watches for changes to this resource and automatically applies the necessary configurations to the underlying logging components.
```

Comment:
It should be said and recommended to create the clusterLogging CR for set limits to the collectors as indicated before and also for being able to set the tolerations. This is reflected in https://issues.redhat.com/browse/OBSDOCS-726.
Then, it should be mentioned the ClusterLogging CR and for what's used